yarn-project/sequencer-client/src/global_variable_builder/global_builder.ts

```diff
@@ -46,8 +46,13 @@ export class GlobalVariableBuilder implements GlobalVariableBuilderInterface {
     });
   }
 
+  /**
+   * Computes the "current" base fees, e.g., the price that you currently should pay to get include in the next block
+   * @returns Base fees for the expected next block
+   */
   public async getCurrentBaseFees(): Promise<GasFees> {
-    return new GasFees(Fr.ZERO, new Fr(await this.rollupContract.read.getManaBaseFee([true])));
+    const timestamp = BigInt((await this.publicClient.getBlock()).timestamp + BigInt(this.ethereumSlotDuration));
```

Comment:
```suggestion
    const timestamp = BigInt((await this.publicClient.getBlock()).timestamp + BigInt(this.aztecSlotDuration));
```
Shouldn't we be moving to the next Aztec slot, as opposed to the next Ethereum one?

Reply:
Made an update I think should handle it better. Now fetching the last block, and ensuring that we are at least at the next block from that. Ensuring that we won't get the fee for the already published block, but for the next possible. Only needed for the end-user, as some of these checks are already done earlier in the run from sequencing.